mod/manifest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+    function manifest_content(App $a) {
+
+		$tpl = get_markup_template('manifest.tpl');
+
+		header("Content-type: application/manifest+json");
```

Comment:
Standards: Please use only one quoting style, it seems single quotes are the most popular.